.gitattributes

```diff
@@ -1,10 +1,11 @@
-.editorconfig          export-ignore
-.gitattributes         export-ignore
+/.editorconfig         export-ignore
```

Comment:
Sorry to be a pain.. but...

https://github.com/guzzle/guzzle/pull/2412#discussion_r352331347

Reply:
ok, I agree if that was on purpose.